src/js/pensions/config/form.js

```diff
@@ -853,9 +830,8 @@ const formConfig = {
                 minItems: 1,
                 items: {
                   type: 'object',
-                  required: ['dependentRelationship', 'fullName'],
+                  required: ['fullName', 'childDateOfBirth'],
                   properties: {
-                    dependentRelationship: dependents.items.properties.dependentRelationship,
```

Comment:
Are there any changes that should be made to the back end schema to reflect this?